### DISCUSSION TITLE
Jaalee Temperature & Humidity Sensor

### DISCUSSION THREAD

**Discussion by muellthosbutler** [custom-components/ble_monitor#1149] [Q&A]
Hi there,

I bought an Jaalee Temperature & Humidity Sensor to monitor my fridge. I bought it over Aliexpress, after I read a comment there, that this is supported by BLE-Monitor Integration. But the device is detected by HA as an "iBeacon tracker" and it displays  a distance in meters, what is obviously incorrect. There is a pull request stating, that temperature an humidity readings are also supported.
Question: How to get these? How to integrate the device properly in HA?

TIA and Best Regards,  muellthos

**Comment by Ernst79** [comment]
Hi, 

Unfortunately, I don't have this sensor myself, but it should send both iBeacon and Jaalee own BLE format data. 

You can try to ask in this issue https://github.com/custom-components/ble_monitor/issues/1042 to see if the owner can help you. 

Otherwise, post some logs here, see the request new sensor page how. https://custom-components.github.io/ble_monitor/sensor_request
